<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentReviewPage</class>
 <widget class="QWidget" name="ComponentReviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Component Review</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QWidget" name="header_widget">
     <layout class="QHBoxLayout" name="horizontalLayout_header">
      <item>
       <widget class="QLabel" name="label_component_name">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Component Name</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_header">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_component_uuid">
        <property name="text">
         <string>&lt;a href="#"&gt;UUID&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="main_splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="component_validation_widget">
      <layout class="QVBoxLayout" name="verticalLayout_component_validation">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_component_validation_header">
         <item>
          <widget class="QLabel" name="label_component_validation_header">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Component Validation Messages</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_component_validation">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_refresh_component_checks">
           <property name="text">
            <string>Refresh Component Checks</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="component_validation_tree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Approval</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Severity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Message</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="device_validation_widget">
      <layout class="QVBoxLayout" name="verticalLayout_device_validation">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_device_validation_header">
         <item>
          <widget class="QLabel" name="label_device_validation_header">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Device Validation Messages</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_device_validation">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_refresh_device_checks">
           <property name="text">
            <string>Refresh Device Checks</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="device_validation_tree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Approval</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Severity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Message</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>